Assets/Scripts/Player/Input Parser/InputParser.cs

```diff
@@ -10,12 +11,16 @@ public sealed class InputParser : MonoBehaviour
         private InputActionAsset _inputActionAsset;
         private PlayerMovement _playerMovement;
 
+        private Camera _mainCamera;
+        private const string INTERACTABLE_TAG = "Interactable";
+
         private void Awake()
         {
             _playerInput = GetComponent<PlayerInput>();
             _inputActionAsset = _playerInput.actions;
             _playerMovement = GetComponent<PlayerMovement>();
-
+            _mainCamera = Camera.main;
```

Comment:
```suggestion
            GetRefrences();
            Init();
```
Wil graag de `Awake` method duidelijk houden, dus graag deze dingen splitsen in nieuwe private functies.